tests/pytests/test_tags.py

```diff
@@ -416,15 +417,15 @@ def testTagAutoescaping(env):
     env.cmd('HSET', '{doc}:22', 'tag', "w'?*1'", 'id', '22')
 
     # Test exact match
-    res = env.cmd('FT.SEARCH', 'idx', '@tag:{abc:1}', 'NOCONTENT',
+    res = env.cmd('FT.SEARCH', 'idx', '@tag:{"abc:1"}', 'NOCONTENT',
```

Comment:
Do we have a test that `|` works inside the`{}`, for example: `@tag:{"foo"|"bar"}`

Reply:
Test included in lines 481-510.